backend/database/seed.sql

```diff
@@ -74,25 +74,25 @@ INSERT INTO brands (email, password, name, business_id)
            ('Nba@gmail.com', 'leberon', 'Espn', 272774721);
 
 INSERT INTO textiles (name, pic, care, environmental_impact)
-    VALUeS ('wool', 'http://localhost:3030/textiles/wool.jpg', 'You shoulc care for wool', 'wool is natural because sheep'),
-          ('polyester', 'http://localhost:3030/textiles/polyester.jpg', 'You shouldnt care for polyester', 'bad for environment, bad for you');
+    VALUeS ('wool', 'http://localhost:3129/textiles/wool.jpg', 'You shoulc care for wool', 'wool is natural because sheep'),
```

Comment:
We will have also to change the link in the database (no http://localhost:3129 )
we will talk later on how we will set the container (public) folder for this
for now it's fine, no changes but we will have to go back to that